### PR TITLE
docs: update env vars acording to CLI output

### PR DESCRIPTION
**Pull request by wtrocki**
Reverts redhat-developer/app-services-guides#468

Reintroduces changs for quickstarts introduced by @rkpattnaik780

### CODE REVIEW

**Comment by wtrocki** [comment]
@carlesarnal FYI. This PR would be blocked till we will be able to configure serdes with OAUTH_TOKEN_ENDPOINT_URI value.

**Comment by carlesarnal** [comment]
#478 PR created to unblock this one

File: docs/kafka/kcat-kafka/README.adoc
Comment (wtrocki) [inline docs/kafka/kcat-kafka/README.adoc:135]: Missing: RHOAS_OAUTH_TOKEN_URL

**Review by rkpattnaik780 — APPROVED** [review]

**Comment by wtrocki** [comment]
We cannot merge this change due to current version of the registry requiring full API path.

We should first review, merge and release: 
https://github.com/redhat-developer/app-services-cli/pull/1539

Making change for the urls in the APP but also change in the guide is needed.

**Comment by wtrocki** [comment]
@rkpattnaik780 We can release RHOAS CLI to unblock this PR form merging

**Comment by rkpattnaik780** [comment]
A new [version](https://github.com/redhat-developer/app-services-cli/releases/tag/v0.41.1) of the CLI has been released. Merging!

File: code-examples/quarkus-service-registry-quickstart/producer/src/main/resources/application.properties
Comment (wtrocki) [inline code-examples/quarkus-service-registry-quickstart/producer/src/main/resources/application.properties:17]: Missing Env var. We should have

mp.messaging.connector.smallrye-kafka.apicurio.registry.url=${SERVICE_REGISTRY_URL}${SERVICE_REGISTRY_CORE_PATH

File: docs/registry/quarkus-registry/README.adoc
Comment (wtrocki) [inline docs/registry/quarkus-registry/README.adoc:144]: We are missing env var here

SERVICE_REGISTRY_CORE_PATH

File: code-examples/quarkus-service-registry-quickstart/producer/src/main/resources/application.properties
Comment (rkpattnaik780) [reply code-examples/quarkus-service-registry-quickstart/producer/src/main/resources/application.properties:17]: Doing it right away!

**Pull request merged**
